Dockerfile.gno.release

```diff
@@ -1,11 +1,14 @@
 FROM busybox
 
-COPY ./gno /gno
+COPY ./gno /usr/bin/gno
 COPY ./examples /gnoroot/examples/
 COPY ./gnovm/stdlibs /gnoroot/gnovm/stdlibs/
 COPY ./gnovm/tests/stdlibs /gnoroot/gnovm/tests/stdlibs/
 
 ENV GNOROOT="/gnoroot/"
 
-ENTRYPOINT [ "/gno" ]
-CMD [ "" ]
\ No newline at end of file
+WORKDIR /gnoroot/
+
```

Comment:
Personal preference for not adding empty lines here. And personal preference for aligning the second word with prepending spaces.